### PR TITLE
Bug 1497253 - Set Bugzilla by default when associating a bug

### PR DESCRIPTION
**Pull request by ionutgoldan**

### CODE REVIEW

File: ui/helpers/errorMessage.js [hunk 1]
```diff
@@ -42,3 +42,18 @@ export const formatTaskclusterError = function formatTaskclusterError(e) {
...
+@param {Error} e error object from Perfherder client.
+*/
+export function alertHttpError(error) {
+    if (error.statusText) {
+        error = 'HTTP Error: ' + error.statusText;
+    }
+    // we could probably do better than print this
+    // rather useless error, but at least this gives
+    // a hint on what the problem is
+    alert('Error loading performance data\n\n' + error);
```
Comment (jmaher) [inline ui/helpers/errorMessage.js]: this helper method is called alertHttpError, but is specific to performance, the comments do help- so I don't know if we need to make this clearer that this is alertPerfherderHttpError

Comment (ionutgoldan) [reply ui/helpers/errorMessage.js]: I think the comments suffice.

Comment (edmorley) [reply ui/helpers/errorMessage.js]: `alert()` is generally frowned upon for non-development, with in-page notifications preferred (eg thNotify, soon to be replaced by something react based for the job view). As such, I think this function should perhaps live in perfherder until its removed later when perfherder is converted to React?

@camd will have a better idea of the future direction, so I'll defer to him :-)

Comment (wlach) [reply ui/helpers/errorMessage.js]: I agree that this should just be left in perfherder for now. I think I only used this function as a stopgap to try and debug some problems.

Comment (ionutgoldan) [reply ui/helpers/errorMessage.js]: I put alertHttpError only in Perfherder. Once thNotify is refactored, I'll replace it with that.

Comment (camd) [reply ui/helpers/errorMessage.js]: Agreed.  PR #4132 (currently in review) will replace ``thNotify`` with a React view component and a ``Context``.  So once you refactor this area to React, it should be easy to use.  For ``helpers`` we generally prefer to throw an exception that is then caught by the calling class which, in turn, reports to the user.

File: ui/js/controllers/perf/graphs.js [hunk 2]
```diff
@@ -686,16 +686,6 @@ perf.controller('GraphsCtrl', [
             }, alertHttpError);
         }
 
-        function alertHttpError(error) {
```
Comment (edmorley) [inline ui/js/controllers/perf/graphs.js]: This function was only used in one place and seems unrelated to the changes in `alerts.js` (which is what this PR was about)?

File: ui/js/services/perf/errors.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import treeherder from '../../treeherder';
+
+treeherder.factory('alertHttpError',
```
Comment (edmorley) [inline ui/js/services/perf/errors.js]: If this move of `alertHttpError()` is being kept in this PR, I would prefer it to be a standard function that's just imported in graphs.js. It doesn't need to be an AngularJS factory.

**Comment by ionutgoldan** [comment]
@edmorley Can we land this one too? It's another feature request from the DevTools team.
